README.md

```diff
@@ -1,2 +1,44 @@
 # docker-liberica-jre
-JRE Docker images using Alpine, Liberica and glibc
+
+## Introduction
+
+Provides base Java Docker images using Alpine, [Liberica] an glibc.
+
+Builds run daily, but updates to images are only pushed if Apline or the JVM is updated.
+
+This repository is derived from the images defined in the
+[Liberica public repo](https://github.com/bell-sw/Liberica) licenced under GPL 2.0.
+
+## Images
+
+This repo currently maintains builds for versions 11 and 17 of the JRE. The image name/tags are as follows:
+
+```
+ghcr.io/permutive-engineering/permutive-jre-liberica:17
+```
+
+```
+ghcr.io/permutive-engineering/permutive-jre-liberica:11
+```
+
+In addition to the major versions, each build of each JRE is also maintained, follow the links below to see details on each version:
```

Comment:
This wording is a bit unclear I think.

Potentially:

> In addition to the major versions, minor versions of each JDK (shipped as a JRE) are maintained individually. Follow the links below to see which minor versions are available:

I think we also need to explain here all the image tags that are available. If minor versions are maintained presumably someone can be more precise than just `permutive-jre-liberica:17`.

Either a link to browse the tags available (similar to dockerhub), or just we explain in a grid here.

Reply:
Oh, they are visible in the front page of the repo in GH... Potentially just a link to that then ;)

Reply:
Oh, "minor" may be the wrong word in my first message, "build" probably was right

Reply:
Thanks, great suggestion. I'll change the text and add a link to the GH packages listing.